src/Tools/dotnet-monitor/CollectionRules/Options/Actions/ExecuteOptionsDefaults.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.TestCommon.Options
+#else
+using System.ComponentModel;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Actions
```

Comment:
Remove the conditional namespaces and just use this one. I think this is cause the build failures.

Reply:
This file also needs to be linked into the SchemaGeneration project.

Reply:
Ah, it would seem this was a case of me not fetching/pulling before pushing. My local branch didn't have all the links in ConfigurationSchema. I'll push again once I confirm things are working locally with the latest changes.